Rooms/masterBedroom.json

```diff
@@ -7,23 +7,48 @@
         "boxes": {
             "triggerSequence": ["boxes", "battery"],
             "loop": "False",
+            "type": "room",
             "0": "There are two moving boxes of stuff underneath the west window.",
             "1": "There are two moving boxes containing a clock, battery, blankets, and clothes.",
             "2": "There are two moving boxes containing a clock, blankets, and clothes."
+        },
+        "clock": {
```

Comment:
You need to add these names to itemDictionary.json or the language parser won't pick them out. ("clock", "sheets")

Reply:
Knew I was forgetting something... lol. Done.